snuba/consumer.py

```diff
@@ -65,15 +69,14 @@ def delivery_callback(self, error, message):
     def flush_batch(self, batch):
         """First write out all new INSERTs as a single batch, then reproduce any
         event replacements such as deletions, merges and unmerges."""
-        processor = enforce_table_writer(self.__dataset).get_stream_loader().get_processor()
         inserts = []
         replacements = []
 
-        for action_type, data in batch:
-            if action_type == processor.INSERT:
-                inserts.append(data)
-            elif action_type == processor.REPLACE:
-                replacements.append(data)
+        for message in batch:
```

Comment:
I couldn't find a convincing reason for allowing a consumer to generate more than one row to store into clickhouse. The only downside is that the batch size we write on Clickhouse may be bigger than the batch we get from the batching consumer.